README.md

```diff
@@ -14,36 +14,52 @@ Testing and benchmarking is done with `googletest` and `google/benchmark`.
 
 ## Getting started
 
-Clone the repository and initialize the submodules (googletest, benchmark).
+The respository is meant to be possible to build "out of the box", with standard
+CMake build procedures.
 
 ```shell
-git clone git@github.com:acts-project/detray.git
-cd detray
-git submodule update --init
+git clone https://github.com/acts-project/detray.git
+cmake -DCMAKE_BUILD_TYPE=Debug -S detray -B detray-build
+cmake --build detray-build
 ```
 
-Running `CMake` (minimal version)
-```shell
-cmake -S . -B <build_directory>
-```
-
-Build
-```shell
-cmake --build <build_directory>
-```
-
-### Plugins
-
-The following algebra plugins are avaiable and can be switched on/off 
-
-| *Plugin* | *CMake Flag* | *Default value* | *Dependency requirement* |
-| ---------|--------------|-----------------|--------------------------|
-| array | DETRAY_ARRAY_PLUGIN | On | - |
-| eigen | DETRAY_EIGEN_PLUGIN | Off | Eigen, http://eigen.tuxfamily.org/ |
-| smatrix | DETRAY_SMATRIX_PLUGIN | Off | ROOT, http://github.com/root-project |
-
-
-
+The following cache variables are available to influence which parts of the
+project would be built:
+
+- `DETRAY_EIGEN_PLUGIN` (`ON` by default), `DETRAY_SMATRIX_PLUGIN`
+  (`OFF` by default), `DETRAY_VC_PLUGIN` (`ON` by default): Boolean
+  flags turning the build of [Eigen](https://eigen.tuxfamily.org),
+  [SMatrix](https://root.cern/doc/master/group__SMatrixGroup.html) and
+  [Vc](https://github.com/VcDevel/Vc) using code on or off.
+  * Note that [Algebra Plugins](https://github.com/acts-project/algebra-plugins)
+    must have all of the appropriate options enabled for whichever option
+    is turned on from these.
+- `DETRAY_IO_CSV`: Boolean option turning on the build of `detray::io_csv`
```

Comment:
Maybe we should explain what it results in. That is turn on the reading of geometry definitions from csv (and build the library for it)